.golangci.yaml

```diff
@@ -144,7 +144,7 @@ issues:
       text: "Error return value of `connector.Delete` is not checked"
 
     # Ignore complicated function that gocritic is suspecting
-    - path: rticonnextdds_connector.go
+    - path: .go
```

Comment:
what is this? We should _path_ it to that very file

Reply:
Now, we run into this issue in multiple files and `.go` seems working for any go files. We can add multiple paths as well if it is better.

Reply:
Maybe it is fine, seems it is a CGO thing anyway.